### ISSUE TITLE
build.sh script does not use buildspec/optionalplugins.pri

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Risehat/freemedforms#372]
```
This issue was created by revision d76583aae172.

build.sh script does not use buildspec/optionalplugins.pri
```

Original issue reported on code.google.com by `eric.mae...@gmail.com` on 30 Jul 2014 at 8:31

**Comment by GoogleCodeExporter** [comment]
```
This issue was updated by revision c430e9f4508c.


- Cleaning code
- better zenity integration
- improved help message
- check requested plugins
- check bundle name (camel case)
```

Original comment by `eric.mae...@gmail.com` on 31 Jul 2014 at 7:17

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 608f80975231.
```

Original comment by `eric.mae...@gmail.com` on 31 Jul 2014 at 7:17
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was updated by revision 0c02f9810d8b.


Reconnecting log file
```

Original comment by `eric.mae...@gmail.com` on 31 Jul 2014 at 7:44
